cmd/update-main_test.go

```diff
@@ -68,9 +68,9 @@ func TestGetCurrentReleaseTime(t *testing.T) {
 		{minioVersion2, minioBinaryPath2, releaseTime2, nil},
 		{"junk", minioBinaryPath2, releaseTime2, nil},
 		{"3.2.0", minioBinaryPath2, releaseTime2, nil},
-		{minioVersion2, "", time.Time{}, fmt.Errorf(errorMessage1)},
-		{"junk", "non-existing-file", time.Time{}, fmt.Errorf(errorMessage2)},
-		{"3.2.0", "non-existing-file", time.Time{}, fmt.Errorf(errorMessage2)},
+		{minioVersion2, "", time.Time{}, errors.New(errorMessage1)},
+		{"junk", "non-existing-file", time.Time{}, errors.New(errorMessage2)},
+		{"3.2.0", "non-existing-file", time.Time{}, errors.New(errorMessage2)},
```

Comment:
You would need to add specific test cases for this PR ie exec name is absolute path.